tripal_jbrowse_mgmt/includes/tripal_jbrowse_mgmt_tracks.form.inc

```diff
@@ -94,11 +94,16 @@ function tripal_jbrowse_mgmt_add_track_form($form, &$form_state, $instance_id) {
     ],
   ];
 
+  $form['data']['symbolic_link'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Symbolic Link'),
+    '#description' => t('Create a symbolic link rather than make a copy of the file.'),
```

Comment:
```suggestion
    '#description' => t('Create a symbolic link rather than make a copy of the file. This only applies when a path on the server is supplied.'),
```